### PR TITLE
Switch RHEL7 to openjdk8

### PR DESCRIPTION
**Pull request by juju4**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -36,3 +39,9 @@
     dest: /etc/profile.d/java_home.sh
     mode: 0644
   when: java_home is defined and java_home != ''
+
+- name: check java version
+  command: java -version
+  changed_when: false
+  register: javaversion
+- debug: var=javaversion
```
Comment (geerlingguy) [inline tasks/main.yml]: Just checking this today—If we want to default to Java 8 on RedHat 7 / CentOS 7, then I'd rather just make it so all the default vars files that are loaded follow the same format (e.g. `{{ ansible_distribution }}-{{ ansible_distribution_version }}`).

Also, I don't want to commit debug statements in the playbook :)

File: tasks/main.yml [hunk 2]
```diff
@@ -7,6 +7,9 @@
   include_vars: "{{ ansible_distribution }}.yml"
   when: ansible_distribution == 'Fedora'
 
+- name: Include version-specific variables for CentOS/RHEL.
+  include_vars: "RedHat-{{ ansible_distribution_version.split('.')[0] }}.yml"
+  when: ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux'
```
Comment (geerlingguy) [inline tasks/main.yml]: Please leave an empty newline between each task/block.

Comment (juju4) [inline tasks/main.yml]: ok. done.

File: tasks/main.yml (hunk 1 above)
Comment (juju4) [inline tasks/main.yml]: there is more discussion here.
I'm usually using
```
include_vars: "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
```
but no good for centos/RHEL
```
include_vars: "RedHat-{{ ansible_distribution_version.split('.')[0] }}.yml"
```
if using ansible_distribution here, it means differentiating RHEL and Centos with identitical content.
As for the split, ansible 2.x introduces ansible_distribution_major_version which might be good for all but not sure. My usual tests are Ubuntu LTS Xenial+Trusty and centos7, sometimes older and debian8.
For now, I left as is on my roles. what is your preferences.

For the debug, agree. but a few times I get caught by having the wrong java and couldn't figure which one without verbose mode or connecting the box :)

**Comment by juju4** [comment]
update? any other change needed? Thanks

**Comment by juju4** [comment]
Thanks!

**Pull request merged**